osu.Game.Tests/Visual/Online/TestSceneBeatmapListingOverlay.cs

```diff
@@ -361,7 +374,13 @@ private void noPlaceholderShown()
             AddUntilStep("\"no maps found\" placeholder not shown", () => !overlay.ChildrenOfType<BeatmapListingOverlay.NotFoundDrawable>().Any(d => d.IsPresent));
         }
 
-        private void setCardSize(BeatmapCardSize cardSize) => AddStep($"set card size to {cardSize}", () => overlay.ChildrenOfType<BeatmapListingCardSizeTabControl>().Single().Current.Value = cardSize);
+        private void setCardSize(BeatmapCardSize cardSize, bool viaConfig) => AddStep($"set card size to {cardSize}", () =>
+        {
+            if (!viaConfig)
```

Comment:
really weird to check a _negated_ flag value - would flip the branches around and not negate this

Reply:
Sure.